examples/src/lib.rs

```diff
@@ -166,3 +195,37 @@ pub enum ExampleType {
         num_signers: usize,
     },
 }
+
+/// Defines a set of available hash function for STARK protocol.
```

Comment:
I would change this to something like: "Defines a set of hash functions available for the provided examples. Some examples may not support all listed hash functions."